velox/exec/tests/TableWriteTest.cpp

```diff
@@ -964,44 +964,36 @@ TEST_P(AllTableWriterTest, renameAndReorderColumns) {
 
 // Runs a pipeline with read + write.
 TEST_P(AllTableWriterTest, directReadWrite) {
-  for (const auto fileFormat :
```

Comment:
Can we make the file format a test parameter so it auto covered by all the table writer test cases? Thanks!

Reply:
@xiaoxmeng Not all the tests are working with Parquet due to https://github.com/facebookincubator/velox/issues/5560.
This PR is currently blocking https://github.com/facebookincubator/velox/pull/5469
Can we merge this to unblock the above PR?

Reply:
I will make this a test parameter.